validation/exclusions_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/ethereum-optimism/superchain-registry/superchain"
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
```

Comment:
Why this? It's best practice to use this very sparingly, mostly only when writing a test of a package `package` in a test package `package_test` for dependency reasons.

Reply:
The `validation` package is essentially a test package for `superchain`, it just happens to be a separate module. I find it a lot neater to do this.